data/SI.slovenia/2007-2013/source.description.yaml

```diff
@@ -0,0 +1,145 @@
+# I am the latest version: use me.
```

Comment:
@seegerseeger Could you remove all the comments before you push? That would make it easier to review! Thanks.